### DISCUSSION TITLE
Research paper on JSBSim

### DISCUSSION THREAD

**Discussion by agodemar** [JSBSim-Team/jsbsim#231] [General]
Dear all,
I believe this is a good place to ask about this subject. If someone feels this is an OT message, I apologize in advance.
If you are involved in a research or project that features flight simulations with JSBSim *and* can consider to (or are allowed to) write down a joint paper on how the software has been determinant to accomplish this or that goal, please let me know.

All sorts of suggestion and criticism are welcome.

As a scholar, I think it's time to produce something valuable that arises from the community of developers and users.

I have created a project named [Research paper on JSBSim](https://github.com/JSBSim-Team/jsbsim/projects/2) in the Projects section.

**Comment by agodemar** [comment]
I am collecting infos on recent use cases of JSBSim.

It would be great to hear how JSBSim has been an effective support for your projects.
Which University or company? Which research group? Which project? Which was your achievement?

I want to mention the most relevant ones in a paper that I have in mind. It will be an updated version of this one: [AIAA-2009-5699_Berndt_DeMarco](http://wpage.unina.it/agodemar/DSV-DQV/AIAA-2009-5699_Berndt_DeMarco.pdf) that was written 10 years ago (2009).

Please, do feel free to contact me if you want to contribute.

**Comment by seanmcleod** [comment]
@agodemar I've sent you an email regarding my use of JSBSim as part of a VSS simulator for test pilot training. Not sure if it will be of enough interest as part of an academic paper.

**Comment by jonsberndt** [comment]
Sean,

I’d be interested to see this, too, if that’s possible.

Jon Berndt

**Comment by seanmcleod** [comment]
@jonsberndt I've just emailed you the info I sent Agostino.

**Comment by agodemar** [comment]
@bcoconni are you willing to appear as a co-author?
JSBSim state of the art means also how we have ported the original project on GitHub and how we manage our CI/CD workflow, how we support users and improve models and code quality.

**Comment by bcoconni** [comment]
> @bcoconni are you willing to appear as a co-author?

Sure, although I am not a researcher myself but a pat in the back is always welcome 😄

**Comment by Gijsrooy** [comment]
Hi Agostino, 

Nice initiative! I will check if some of my colleagues at Delft University of Technology are interested in contributing. In fact, I'm currently at AIAA's SciTech in Orlando, trying to attend all of the FlightGear/JSBsim presentations. Will see if I can get some additional contacts from that.

Gijs

**Comment by agodemar** [comment]
Hi Gijs, yes you're in the perfect spot to get an idea of recent/advanced JSBSim use cases.
I'd be great if you could report to us (or in DM) your impressions.

**Comment by jonsberndt** [comment]
How many papers is FlightGear and JSBSim mentioned in?

Jon

**Comment by Gijsrooy** [comment]
Hi Jon,

There are 3 papers with JSBSim being mentioned (and actively used), 2 of those also used FlightGear, bringing the grand total for FlightGear on 7 papers this week.

**Comment by seanmcleod** [comment]
@Gijsrooy can you post the abstracts of the 7 papers here? When I looked at AIAA's conference website I could only find the title of the talk/paper. I'm assuming as an attendee you might have online access to the list of abstracts?

**Comment by Gijsrooy** [comment]
You can find the abstracts at ARC (they are free to access, but for the actual papers you must have a subscription):

- https://arc.aiaa.org/doi/10.2514/6.2020-0135
- https://arc.aiaa.org/doi/10.2514/6.2020-0136
- https://arc.aiaa.org/doi/10.2514/6.2020-0373
- https://arc.aiaa.org/doi/10.2514/6.2020-0409
- https://arc.aiaa.org/doi/10.2514/6.2020-0859
- https://arc.aiaa.org/doi/10.2514/6.2020-1401
- https://arc.aiaa.org/doi/10.2514/6.2020-1968
- https://arc.aiaa.org/doi/10.2514/6.2020-1970

**Comment by seanmcleod** [comment]
@Gijsrooy perfect thanks!

**Comment by seanmcleod** [comment]
And @Gijsrooy I see you authored one of the papers - https://arc.aiaa.org/doi/10.2514/6.2020-0373, although I can only read the abstract at the moment ;<

**Comment by Gijsrooy** [comment]
That's correct Sean! I presented my master thesis this Monday: https://forum.flightgear.org/viewtopic.php?f=34&t=35902 I'm afraid we only used FlightGear for the visuals though, so no JSBSim paper there ;-)

**Comment by seanmcleod** [comment]
@Gijsrooy congrats. Which programmable sidestick did you use? I've used one from Stirling Dynamics in the past and currently using the CLS-P joystick from Brunner.

**Comment by Gijsrooy** [comment]
In our experiment we used a (rather old) Moog FCS Ecol-8000, that allows us to adjust the forces in the stick dynamically.

**Comment by bcoconni** [comment]
This issue has been moved to "Discussions". Now that this new feature is available in GitHub, it has been decided to move issues related to support requests and questions to "Discussions" and restrict "Issues" to bugs and feature requests.
This change is only motivated by project organization and will not affect the support provided. We'll gladly continue to answer your questions 😄.
